docs/development_suite/set-up-infrastructure/configure-provider.mdx

```diff
@@ -206,13 +206,174 @@ A modal will appear allowing you to select one of the providers implementing tha
 
 Once you edit the default provider, you will be able to see the card with the new provider set.
 
-## Associate a Provider to a Project
+## Associate Providers to Projects
 
-If you haven't specified a default provider on a certain capability, or you wish to overwrite the default provider configuration, you will need to manually associate the provider with the projects you want to use it for within that company.
+If you haven't specified a default provider on a certain capability, or you wish to overwrite the default provider configurations on a specific project within the company, you will need to manually associate the provider with the project you want to use it for.
 
-To do this, you can use the CMS: navigate through the `Projects` section, select the desired project, and change the `providerId` field value in the `Repository` object.
+To do this, you can use the CMS: navigate through the `Projects` section, select the desired project, and edit the document by following the guidelines below, which specify what information to include depending on the provider type and capability.
 
-![Associate provider to project CMS](./img/associate-provider-to-project-cms.png)
+### Edit Git Provider
+
+To change the Git Provider of a project, simply edit the `providerId` field within the `Repository` object by entering the `providerId` (obtainable from the providers' CRUD collection) you want to use instead of the previous one.
```

Comment:
Come ottiene l'utente il "providerId" del provider che deve inserire? Dove lo può trovare?

Reply:
Lo ottiene appunto andando a guardare il documento nella collezione CRUD dei provider. In Console ad oggi non viene visualizzato da nessuna parte 😬

Reply:
Ma dov'è quella collezione "providers" nel CRUD collection? Perchè è questo che manca. A quale collezione ci stiamo riferendo? Perchè parliamo del CRUD e viene da pensare riferito al progetto, ma sicuramente non può essere il CRUD di quel progetto

Reply:
Non so bene come risponderti, nel senso che "providers" è una collezione sul CRUD alla pari di "projects". Quindi semplicemente l'utente va sul suo CRUD a guardarsi il documento del provider che vuole sostituire a quello vecchio, si piglia il `providerId` e poi lo schianta qua